test/build-targets-test.js

```diff
@@ -60,9 +60,25 @@ test('build targets using Node 10 (ABI 64)', function (t) {
 
 test('build targets using Node 11 (ABI 67)', function (t) {
   t.plan(1)
-  const targets = buildTargets('67', supportedTargets)
+  const targets = buildTargets('67', supportedTargets, {})
   t.deepEqual(targets, [
     { runtime: 'node', abi: '67' },
     { runtime: 'electron', abi: '69' }
   ])
 })
+
+test('build napi targets when napi_versions found', function (t) {
+  t.plan(1)
+  const targets = buildTargets('67', supportedTargets, {
+    binary: {
+      napi_versions: [3, 4]
+    },
+    dependencies: {
+      'node-addon-api': '^1.6.3'
+    }
+  })
+  t.deepEqual(targets, [
+    { runtime: 'napi', abi: '3' },
+    { runtime: 'napi', abi: '4' }
+  ])
```

Comment:
Why/when would you need > 1 build (if not to separate runtime, by which I mean node and electron)?

Reply:
There are currently 4 `napi` ABI versions - see https://nodejs.org/dist/latest-v12.x/docs/api/n-api.html#n_api_n_api_version_matrix - and  there is not a 1:1 mapping between these and `node` ABIs.

Reply:
N-API is forwards-compatible, so you only need one build.

Reply:
I'd like to be able to offer support for Node 8.10.0 on AWS Lambda (napi ABI 2) but also allow users of Node 10.x to take advantage of newer features such as BigInt (napi ABI 3). The `napi_versions` property is an array in prebuild so this approach at least keeps things consistent.